src/misc/day.ts

```diff
@@ -0,0 +1,24 @@
+const TICKS = 2;
+const NIGHT_BEGIN = 13000;
+const NIGHT_END = 23000;
+let count = 0;
```

Comment:
More descriptive name would help. Additionally, this could be placed immediately before setInterval instead of immediately after constants.